ui/components/component-library/sensitive-text/sensitive-text.types.ts

```diff
@@ -0,0 +1,16 @@
+import type { TextProps } from '../text/text.types';
+
+export enum SensitiveLengths {
+  Short = 5,
+  Medium = 8,
+  Long = 13,
```

Comment:
I am going to change these integer enums into string versions based on the feedback from mobile
https://github.com/MetaMask/metamask-mobile/pull/11965#discussion_r1813222495